packages/flutter/lib/src/material/outline_button.dart

```diff
@@ -0,0 +1,504 @@
+///  * [FloatingActionButton], the round button in material applications.
+///  * [IconButton], to create buttons that just contain icons.
+///  * [InkWell], which implements the ink splash part of a flat button.
+///  * <https://material.google.com/components/buttons.html>
+class OutlineButton extends StatefulWidget {
+  /// Create a filled button.
+  ///
+  /// The [highlightElevation], and [borderWidth]
+  /// arguments must not be null.
+  const OutlineButton({
+    Key key,
+    @required this.onPressed,
+    this.textTheme,
+    this.textColor,
+    this.disabledTextColor,
+    this.color,
+    this.highlightColor,
+    this.splashColor,
+    this.highlightElevation: 2.0,
+    this.borderSide,
+    this.disabledBorderColor,
+    this.highlightedBorderColor,
+    this.padding,
+    this.shape,
+    this.child,
+  }) : assert(highlightElevation != null && highlightElevation >= 0.0),
+       super(key: key);
+
+  /// Create an outline button from a pair of widgets that serve as the button's
+  /// [icon] and [label].
+  ///
+  /// The icon and label are arranged in a row and padded by 12 logical pixels
+  /// at the start, and 16 at the end, with an 8 pixel gap in between.
+  ///
+  /// The [highlightElevation], [icon], and [label] must not be null.
+  OutlineButton.icon({
+    Key key,
+    @required this.onPressed,
+    this.textTheme,
+    this.textColor,
+    this.disabledTextColor,
+    this.color,
+    this.highlightColor,
+    this.splashColor,
+    this.highlightElevation: 2.0,
+    this.borderSide,
+    this.disabledBorderColor,
+    this.highlightedBorderColor,
+    this.shape,
+    @required Widget icon,
+    @required Widget label,
+  }) : assert(highlightElevation != null && highlightElevation >= 0.0),
+       assert(icon != null),
+       assert(label != null),
+       padding = const EdgeInsetsDirectional.only(start: 12.0, end: 16.0),
+       child = new Row(
+         mainAxisSize: MainAxisSize.min,
+         children: <Widget>[
+           icon,
+           const SizedBox(width: 8.0),
+           label,
+         ],
+       ),
+       super(key: key);
+
+  /// Called when the button is tapped or otherwise activated.
+  ///
+  /// If this is set to null, the button will be disabled, see [enabled].
+  final VoidCallback onPressed;
+
+  /// Defines the button's base colors, and the defaults for the button's minimum
+  /// size, internal padding, and shape.
+  ///
+  /// Defaults to `ButtonTheme.of(context).textTheme`.
+  final ButtonTextTheme textTheme;
+
+  /// The color to use for this button's text.
+  ///
+  /// The button's [Material.textStyle] will be the current theme's button
+  /// text style, [ThemeData.textTheme.button], configured with this color.
+  ///
+  /// The default text color depends on the button theme's text theme,
+  /// [ButtonThemeData.textTheme].
+  ///
+  /// See also:
+  ///   * [disabledTextColor], the text color to use when the button has been
```

Comment:
you need a blank line before the start of each bulleted list for dartdoc to do the right thing. (applies across the file)

Reply:
Fixed in https://github.com/flutter/flutter/pull/15092